App/styles/typography.js

```diff
@@ -0,0 +1,14 @@
+import { StyleSheet } from 'react-native';
+
+export const typography = StyleSheet.create({
+  default: {
+    fontFamily: 'WorkSans_400Regular',
+  },
+  bold: {
+    fontFamily: 'WorkSans_700Bold',
+  },
+});
+
```

Comment:
Use same font that are in which are in design

Reply:
Font mention in figma is paid